docs/speech-usage.rst

```diff
@@ -97,6 +97,7 @@ Great Britian.
      transcript: Hello, this is one test
      confidence: 0
 
+
```

Comment:
Something weird happened when I rebased upstream master into it. Removed in the next push.